.github/workflows/ci.yml

```diff
@@ -225,5 +225,15 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20
-      - run: npm install
-      - run: npm run lint
+
+      - name: Setup NPM dependencies
+        run: npm install
+
+      - name: Lint i18n
+        run: npm run lint:i18n
+
+      - name: Lint styles
+        run: npm run lint:styles
+
+      - name: Lint JS
+        run: npm run lint:js
```

Comment:
I wonder if it is better to have a task for each of those linting checks

Reply:
In the past we also had all of these in one CI task, except we used the old Grunt config where there was a single `npm run test`, which forced it into a single task.

The main pain point is `npm install` being slow (in theory). But it might not necessarily be an issue here.